extmod/modssl_mbedtls.c

```diff
@@ -567,6 +859,9 @@ STATIC const mp_rom_map_elem_t ssl_socket_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_ioctl), MP_ROM_PTR(&mp_stream_ioctl_obj) },
     #endif
     { MP_ROM_QSTR(MP_QSTR_getpeercert), MP_ROM_PTR(&mod_ssl_getpeercert_obj) },
+    #ifdef MICROPY_SSL_MBEDTLS_EXTRAS
+    { MP_ROM_QSTR(MP_QSTR_cipher), MP_ROM_PTR(&mod_ssl_cipher_obj) },
```

Comment:
What is this method needed for?

Reply:
To return the current TLS protocol and ciphersuite being used see, https://docs.python.org/3/library/ssl.html#ssl.SSLSocket.cipher, 
CPython returns a 3-element tuple, but in `mbedtls` there is no function for the third element .i.e 
> the number of secret bits being used

But I think it's useful enough even if it doesn't match CPython and I don't think it would break any compatibility